docs/usage/getting-started/private-packages.md

```diff
@@ -322,7 +322,19 @@ npmRegistries:
 ### nuget
 
 For each known NuGet registry, Renovate searches for `hostRules` with `hostType=nuget` and matching host.
-For those found, a command similar to the following is run: `dotnet nuget add source ${registryInfo.feedUrl} --configfile ${nugetConfigFile} --username ${username} --password ${password} --store-password-in-clear-text`
+For those found, a command similar to the following is
+run: `dotnet nuget add source ${registryInfo.feedUrl} --configfile ${nugetConfigFile} --username ${username} --password ${password} --store-password-in-clear-text`
+
+```js
+hostRules: [
+  {
+    matchHost: 'https://pkgs.dev.azure.com/<org>/',
+    hostType: 'nuget',
+    username: 'user',
+    password: '<Encrypted PAT Token>',
```

Comment:
what means `Encrypted` here? If it's renovate encryption, it needs to  be
```suggestion
    encryped: { password: '<Encrypted PAT Token>' },
```

Reply:
@viceice i copied from the discussion https://github.com/renovatebot/renovate/discussions/10998
and from other examples

Reply:
```suggestion
    password: '<PAT>',
```

https://github.com/renovatebot/renovate/discussions/10998#discussioncomment-1086730